control/control.CASP.xml

```diff
@@ -362,99 +362,28 @@ defined by a role can be overridden in the next steps if necessary.&lt;/p&gt;</l
             <name>initial</name>
             <unique_id>inst_initial</unique_id>
             <enable_skip>no</enable_skip>
-
             <proposal_modules config:type="list">
                 <proposal_module>
-                    <name>clone</name>
-                    <presentation_order>95</presentation_order>
-                </proposal_module>
-                <proposal_module>
-                    <name>hwinfo</name>
-                    <presentation_order>90</presentation_order>
+                    <name>software</name>
+                    <presentation_order>10</presentation_order>
+                    <read_only config:type="boolean">true</read_only>
                 </proposal_module>
                 <proposal_module>
                     <name>partitions</name>
```

Comment:
I think that partitions and bootloader should be swapped (according to the trello card).